Resources/Prototypes/_Mono/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/projectiles.yml

```
@@ -33,3 +33,26 @@
       intensitySlope: 3
       totalIntensity: 40
       maxTileBreak: 1
+
+- type: entity
+  id: LightPlasmaProjectile
+  name: plasma projectile
+  parent: BaseBullet
+  categories: [ HideSpawnMenu ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)